test/dict.jl

```diff
@@ -1247,3 +1247,16 @@ end
 let c = bar()
     @test c === missing || c == ComparesWithGC38727(1)
 end
+
+@testset "Detect multiple concurent writes" begin
+    if Threads.nthreads() > 1
+        @test_throws TaskFailedException let d = Dict()
+            Threads.@threads for i = 1:1000
+                d[i] = i
+            end
+        end
+    else
+        # Multiple threads not enabled.
+        @test false
```

Comment:
Not sure, if multiple threads are enabled in CI. Just to make sure.

Reply:
Any idea on how to enable CI with `Threads.nthreads() != 1`? Going through other tests without and not finding the right way.

Reply:
Does it mean that the following test is disabled now in CI? https://github.com/JuliaLang/julia/blob/897c3cfdda888c3149724531e07cdd4f0fede895/test/threads_exec.jl#L30-L32